src/main.jsx

```diff
@@ -36,41 +36,27 @@ const router = createBrowserRouter(
       <Route path="blog" element={<BlogPage />} />
       <Route path="blog/:slug" element={<Blogs />} />
       <Route path="/book-nurse" element={<Booking />} />
-      <Route path="privacypolicy" element={<PrivacyPolicy />} />
-      <Route path="licensing" element={<Licensing />} />
-      <Route path="termsandconditions" element={<TermsAndConditions />} />
+      <Route path="/privacypolicy" element={<PrivacyPolicy />} />
+      <Route path="/guidelines" element={<Guidelines/>}/>
+      <Route path="/licensing" element={<Licensing />} />
+      <Route path="/termsandconditions" element={<TermsAndConditions />} />
     </Route>
   )
 );
 
 export const ThemeContext = createContext(null);
 
 const App = () => {
-  const [theme, setTheme] = useState(() => {
-    const storedTheme = localStorage.getItem("theme");
-    return storedTheme || "light";
-  });
+  const [theme, setTheme] = useState("light");
 
   const toggleTheme = () => {
-    setTheme((curr) => {
-      const newTheme = curr === "light" ? "dark" : "light";
-      localStorage.setItem("theme", newTheme);
-      return newTheme;
-    });
+    setTheme((curr) => (curr === "light" ? "dark" : "light"));
   };
 
   useEffect(() => {
     document.body.className = theme;
   }, [theme]);
 
-  useEffect(() => {
-    const lenis = new Lenis();
-    function raf(time) {
-      lenis.raf(time);
-      requestAnimationFrame(raf);
-    }
-    requestAnimationFrame(raf);
-  }, []);
```

Comment:
@Hemashree21 Ah. This is not the intended change right ? You don't mean to delete this part of code. And so do other parts like the route etc. so what you can do is copy your current main branch into another new branch temporarily.

```
git checkout -b temp
```

Now, Come back to your demo branch

```
git checkout demo
```

Now bring the head of demo exactly with the main of upstream repo. But before that make sure you have already added upstream in your git 
```
git remote add upstream https://github.com/Nactore-Org/Nacto-Care.git
```
(If you are using SSH use the SSH instead of HTTPS URL)

Now reset demo branch with main of upstream
```
git reset --hard upstream/main
```

Make sure at this point there are no commits of yours present in the demo branch and the commit log is exactly as same as the main of Nacto-Care.

Now use cherry-pick as below for three of your commits in order and add them as shown below

```
git cherry-pick <commit-hash>
```

Finally verify with git log that your three commits have been added or not. 

If you still face any problems do let me know

Reply:
The temp branch is just for backup in case you lose up your changes while reseting or any merge conflicts occurs.